mode/libraries/vr/build.gradle

```diff
@@ -13,43 +13,54 @@ dependencies {
-       project {
-           groupId "org.p5android"
-           artifactId "processing-vr"
-           version "${vrLibVersion}"
-           packaging "jar"
-           licenses {
-               license {
-                   name "GNU Lesser General Public License, version 2.1"
-                   url "https://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt"
-                   distribution "repo"
-               }
-           }
-           dependencies {
-               dependency {
-                   groupId "org.p5android"
-                   artifactId "processing-core"
-                   version "${modeVersion}"
-                   scope "implementation"
-               }
-
-               dependency {
-                   groupId "com.google.vr"
-                   artifactId "sdk-base"
-                   version "${gvrVersion}"
-                   scope "implementation"
-               }
-               dependency {
-                   groupId "com.google.vr"
-                   artifactId "sdk-audio"
-                   version "${gvrVersion}"
-                   scope "implementation"
-               }               
-           }
+task sourceJar(type: Jar, dependsOn: classes) {
+    classifier = "sources"
+    from sourceSets.main.allSource
+}
+
+publishing {
+    publications {
+        vrPublication(MavenPublication) {
+            from components.java
+            artifact sourceJar
+            pom {
+                groupId = "org.p5android"
+                artifactId = "processing-vr"
+                version = "${vrLibVersion}"
+                packaging = "jar"
+                licenses {
+                    license {
+                        name = "GNU Lesser General Public License, version 2.1"
+                        url = "https://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt"
+                        distribution = "repo"
+                    }
+                }
+            }
+
+            pom.withXml {
+                // inserting the dependencies node
+                def dependenciesNode = asNode().appendNode('dependencies')
+                // start adding dependency nodes inside dependencies node
+                def dependencyNode = dependenciesNode.appendNode('dependency')
+                dependencyNode.appendNode('groupId', 'org.p5android')
+                dependencyNode.appendNode('artifactId', 'processing-core')
+                dependencyNode.appendNode('version', "${modeVersion}")
+                dependencyNode.appendNode('scope', 'implementation')
+
+                def dependencyNode2 = dependenciesNode.appendNode('dependency')
+                dependencyNode2.appendNode('groupId', 'com.google.vr')
+                dependencyNode2.appendNode('artifactId', 'sdk-base')
+                dependencyNode2.appendNode('version', "${gvrVersion}")
+                dependencyNode2.appendNode('scope', 'implementation')
+
+                def dependencyNode3 = dependenciesNode.appendNode('dependency')
```

Comment:
Please rename variables here as well !

Reply:
Corrected variables in VR build.gradle file.